From 062988d7c411a04003b70e29b9a07649ca1583d9 Mon Sep 17 00:00:00 2001
From: Andrei Karas <akaras@inbox.ru>
Date: Fri, 17 Dec 2010 03:10:34 +0200
Subject: [PATCH 110/226] Fix equipment sprites show bug.

If no shoes equiped, server dont show cap and gloves slot items.
---
 src/map/clif.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/map/clif.c b/src/map/clif.c
index aa01fe4..c0234f1 100644
--- a/src/map/clif.c
+++ b/src/map/clif.c
@@ -2684,8 +2684,7 @@ int clif_changelook_towards (struct block_list *bl, int type, int val,
             int  equip_point = equip_points[type];
 
             WBUFB (buf, 6) = type;
-            if (sd->equip_index[equip_point] >= 0 && sd->equip_index[2] < MAX_INVENTORY
-                && sd->inventory_data[sd->equip_index[2]])
+            if (sd->equip_index[equip_point] >= 0)
             {
                 if (sd->
                     inventory_data[sd->equip_index[equip_point]]->view_id > 0)
-- 
2.1.0

